doc/source/structures/communication/connection.rst

```diff
@@ -0,0 +1,76 @@
+.. _connection:
+
+Connection
+==========
+
+Connections represent your ability to communicate with other :struct:`processors <kOSProcessor>` or :struct:`vessels <Vessel>`. You can
+use them to find out whether such communication is currently possible and send messages.
+
+Obtaining a connection
+----------------------
+
+There are 2 types of connections. The first type is used to communicate with other :struct:`processors <kOSProcessor>` within the same vessel.
+You can obtain a connection by using `CONNECTION` suffix. Assuming your vessel has a second processor tagged `'second'` it could look like this::
+
+  SET PROCESSOR TO PROCESSOR("second").
+  SET CONNECTION TO PROCESSOR:CONNECTION.
```

Comment:
Does this actually work?  (Using the same identifier name "processor" as both the name of a built-in function AND the name of a user variable.)  Even if it does, I think it would help avoid confusion if the example didn't use the same name for both.